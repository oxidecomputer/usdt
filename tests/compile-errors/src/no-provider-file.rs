#![feature(asm)]

usdt::dtrace_provider!("non-existent.d");

fn main() { }
