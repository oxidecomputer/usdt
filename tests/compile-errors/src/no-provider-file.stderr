error: proc macro panicked
 --> $DIR/no-provider-file.rs:3:1
  |
3 | usdt::dtrace_provider!("non-existent.d");
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Could not read D source file "non-existent.d" in "$WORKSPACE/target/tests/compile-errors": Os { code: 2, kind: NotFound, message: "No such file or directory" }
